montage/loaders.py

```diff
@@ -182,18 +184,25 @@ def get_entries_from_gsheet(raw_url, source='local'):
 
 
 def load_by_filename(filenames, source='local'):
-    ret = []
+    entries = []
+    warnings = []
     if source == 'remote':
         files, warnings = get_by_filename_remote(filenames)
     else:
         files = []
         for filename in filenames:
             file_info = get_file_info(filename)
-            files.append(file_info)
+            if file_info is not None:
+                files.append(file_info)
+            else:
+                warnings.append(
+                    'file "%s" does not exist, please check that its name is spelled correctly, '
+                    'that it has not been renamed or removed' % (filename,)
```

Comment:
This is a good, actionable error message. However, we should keep an eye out for if there are many, many missing files. Assuming there are 0-10, I think the error display method defined in the JS above should be fine.